src/views/FindOrder.vue

```diff
@@ -314,10 +428,33 @@ export default defineComponent ({
       ).then(() => {
         event.target.complete();
       })
+    },
+    async openOrderFilterModal() {
+      const orderFilterModal = await modalController.create({
+        component: OrderFilterModal
+      });
+      return orderFilterModal.present();
+    },
+    orderCreationDateUpdated(ev: CustomEvent) {
+      this.appliedFilters.date.orderCreated = ev['detail'].value
+      this.getOrders();
+    },
+    orderAutoCancelDateUpdated(ev: CustomEvent) {
+      this.appliedFilters.date.autoCancelDate = ev['detail'].value
+      this.getOrders();
+    },
+    orderPromiseDateUpdated(ev: CustomEvent) {
```

Comment:
```suggestion
    updateOrderPromiseDate(ev: CustomEvent) {
```
Same applies to other methods

Reply:
Sir, changed this logic to use a single method `appliedFiltersUpdated` which in turn calls an action that prepares a query for us. #71